charmcraft/templates/init-simple/charmcraft.yaml.j2

```diff
@@ -1,14 +1,11 @@
-# Copyright {{ year }} {{ author }}
-# See LICENSE file for licensing details.
+# This file configures Charmcraft.
+# See https://juju.is/docs/sdk/charmcraft-config for guidance.
 
-# Learn more about charmcraft.yaml configuration at:
-# https://juju.is/docs/sdk/charmcraft-config
-#
-type: "charm"
+type: charm
 bases:
   - build-on:
-    - name: "ubuntu"
+    - name: ubuntu
       channel: "20.04"
```

Comment:
are we not ready for 22.04?

Reply:
@jnsgruk any particular detail why we're not in Jammy here?

Reply:
Good question, I'd go with jammy.

I've not confirmed 💯 that it'll deploy, but pretty sure it will. Might just be worth checking.

Reply:
Checking everything, including deployment.

Reply:
Oh, the template was broken (bad `metadata.yaml`), but after the fix it deploys ok!

FTR, with this diff:
```diff
$ git diff --no-ext-diff
diff --git a/charmcraft/templates/init-simple/charmcraft.yaml.j2 b/charmcraft/templates/init-simple/charmcraft.yaml.j2
index 40eb799d..900d34a2 100644
--- a/charmcraft/templates/init-simple/charmcraft.yaml.j2
+++ b/charmcraft/templates/init-simple/charmcraft.yaml.j2
@@ -5,7 +5,7 @@ type: charm
 bases:
   - build-on:
     - name: ubuntu
-      channel: "20.04"
+      channel: "22.04"
     run-on:
     - name: ubuntu
-      channel: "20.04"
+      channel: "22.04"
diff --git a/charmcraft/templates/init-simple/metadata.yaml.j2 b/charmcraft/templates/init-simple/metadata.yaml.j2
index 6b7eaab5..43a751c5 100644
--- a/charmcraft/templates/init-simple/metadata.yaml.j2
+++ b/charmcraft/templates/init-simple/metadata.yaml.j2
@@ -28,7 +28,7 @@ description: |
 # Your workload’s containers.
 containers:
   httpbin:
-    resource: httpbin
+    resource: httpbin-image
 
 # This field populates the Resources tab on Charmhub.
 resources:
```